### ISSUE TITLE
Image Compiler

### ISSUE THREAD

**Issue by LemmondDesign** [hiukim/mind-ar-js#355]
I paid for the Udemy tutorial and I’m developing locally with MAMP. 

I followed every step with code, but I can’t seem to get my camera to identify a new image I’ve compiled.

do you have any advice?

**Comment by hiukim** [comment]
Do you mean the course example works, but only when you compile your own target image and use your own .mind file it fails?

**Comment by LemmondDesign** [comment]
I mean the opposite. I’ve run several images through the compiler, but when I use your code with a name change for the compiler, it doesn’t work. I’ve even tried compiling an inverted version of the soft mind image.

**Comment by LemmondDesign** [comment]
Perhaps I should clarify. I downloaded the exercise files from Udemy and installed them on my localhost (MAMP). Then I compiled an image using your tool in the MindAR docs (https://hiukim.github.io/mind-ar-js-doc/tools/compile). I placed the newly created targets.mind file in the targets folder and changed "/course-banner.mind" to "/targets.mind" in main.js. Yet, my image doesn't trigger the green-blue plane. However, the image you use in the tutorial still triggers the plane. I feel like I'm so close! Thanks. I can provide images and file if that helps.
